openstack/sre/templates/prometheus-alerts.yaml

```diff
@@ -13,24 +13,65 @@ spec:
   groups:
   - name: openstack-sre.alerts
     rules:
-    - alert: ApiAvailabilityWarning
+    # quick availability check - target updates
+    - alert: ApiAvailabilityWarning1Hour
       expr: (1 - global:api_errors_per_request_sli:ratio_rate1h) *100 <= on (ingress,region) global:api_availability_warning_slo:percent
       labels:
         context: availability
         service: "{{`{{$labels.ingress}}`}}"
         severity: warning
         tier: sre
+        playbook: 'docs/devops/alert/sre-alerts.html'
       annotations:
-        summary: "An API fell below availability warning SLO in the last hour"
-        description: "{{`{{$labels.ingress}}`}} fell below availability warning SLO in the last hour."
+        summary: "API availability warning SLO hit: 1h"
+        description: "{{`{{$labels.ingress}}`}} fell below availability warning SLO in the last hour (default:  *99.8%*)."
```

Comment:
This makes me think, that we have to create one alert per service we defined, so we can show the actual value instead of just a default. This would also help in later on routing the alerts to the right channels, e.g. cc-os-nova for the nova alerts.

Reply:
Mmh .. you know what that I didn't found a way to put `(ingress,region) global:api_availability_critical_slo:percent` into the text sucks -  don't like this double definition.

But I agree on your proposal - just need to add the service and it should be routed than -- maybe some relabel magic from ingress name? 
Should we do it from start on?

Reply:
I assumed we could use our chart's values and with that have both service-name and the actual value we also fed to prometheus. We can then use both to build the alert - each with their own name e.g. `NovaAvailabilitySucksAlert` from `{{service}AvailabilitySucksAlert`. Don't know how to capitalize with go-template, yet, but we have something like that for other alerts too, don't we?

Reply:
true - I put it into the todo